content/en/docs/Modeling/IntConstraints.md

```diff
@@ -7,14 +7,105 @@ description: >
 ---
 
 ## Arithmetical constraints
+These constraints are used to express arithmetical relationships between two or three variables, or between one or two variables and an integer constant.
+For instance, a threshold constraint can be posted as following:
 
+```java
+model.arithm(x, ">=", 3).post();
+```
+
+Accepted mathematical operators for these constraints are `>=`, `>`, `<=`, `<`, `=` and `!=`. When more than one variable is constrained, one can use the following methods:
+
+```java
+String op = ">="; // among ">=", ">", "<=", "<", "=" and "!="
+model.arithm(x, op, y).post();
+// with x an IntVar and y either an IntVar or an int
+
+String op2 = "+"; // either "+" or "-"
+model.arithm(x, op2, y, op, z).post();
+// with x and y IntVar, and z is either an int or an IntVar
+```
+
+## Other mathematical constraints
+Choco-solver also supports other mathematical constraints than arithmetical ones, such as *times*, *div*, *modulo*... In this section, we will consider `x`, `y` and `z` to be `IntVar` and `a` and `b` to be `int`.
+
+### Binary and ternary constraints
+
+*times* constraints are posted as following:
+
+```java
+model.times(x, a, z).post(); // x * a = z
+model.times(x, y, a).post(); // x * y = a
+model.times(x, y, z).post(); // x * y = z
+```
+
+{{% alert title="Info" color="primary" %}}
+For the first type of *times* constraint, one might prefer to use a view: `z = model.intOffsetView(x, a);`.
+{{% /alert %}}
+
+A *div* constraint corresponds to a euclidean division. It is posted as following:
+
+```java
+model.div(x, y, z).post(); // x / y = z
+// it assures that y != 0, and z is rounding towards 0.
+```
+
+*modulo* constraints are of three types:
+
+```java
+model.mod(x, a, b).post(); // x % a = b
+model.mod(x, a, z).post(); // x % a = z
+model.mod(x, y, z).post(); // x % y = z
+```
 
+An *absolute* constraint is posted like this:
+
+```java
+model.absolute(x, y).post(); // x = |y|
+```
+
+{{% alert title="Info" color="primary" %}}
```

Comment:
Actually, there is no such view: it directly  create a `absolute` constraint. This alert can be removed.